.github/workflows/ci.yaml

```diff
@@ -33,22 +33,22 @@ jobs:
         env:
           # we keep 18.0.2 for backwards compatibility with RH-SSO 7.6
           - KEYCLOAK_VERSION: 18.0.2
-          - KEYCLOAK_VERSION: 21.1.1
-          - KEYCLOAK_VERSION: 22.0.4
+          - KEYCLOAK_VERSION: 21.1.2
+          - KEYCLOAK_VERSION: 22.0.5
```

Comment:
@jonasvoelcker since only the tests fail with this 22.0.5 version can you maybe give it a try with the old version?